src/configuration.cc

```diff
@@ -68,6 +68,7 @@ constexpr const char kKubernetesDefaultClusterLocation[] = "";
 constexpr const char kKubernetesDefaultNodeName[] = "";
 constexpr const bool kKubernetesDefaultUseWatch = true;
 constexpr const bool kKubernetesDefaultClusterLevelMetadata = false;
+constexpr const bool kKubernetesDefaultServiceMetadata = false;
```

Comment:
[Optional] Now that you have an additional guard, this can default to `true`...

Reply:
since the cluster level flag is already false, prefer to have it be the same.

Reply:
This would mean that users would have to enable cluster-level watches *and* the service watches. I would have thought they'd want services by default if they're running at cluster level...
Note: I just turned `KubernetesUseWatch` off by default, so that's an additional guard, too. So, back to the original question: if a customer runs the agent and explicitly turns on both `KubernetesUseWatch` and `KubernetesClusterLevelMetadata`, should they expect to see service metadata? I would guess yes.

Reply:
done